addons/report/models/report.py

```diff
@@ -420,7 +420,9 @@ def _run_wkhtmltopdf(self, headers, footers, bodies, landscape, paperformat, spe
             pass
 
         # Wkhtmltopdf arguments
-        command_args.extend(['--quiet'])  # Less verbose error messages
+        # --quiet: Less verbose error messages
+        # --print-media-type: Use stylesheets for printers defined using @media print
+        command_args.extend(['--print-media-type', '--quiet'])
```

Comment:
@tivisse 
without this fix, <a href="https://github.com/odoo/enterprise/pull/801/files#diff-75177802408307e12253b1cc5e23d4c4R63">these</a> properties won't be considered by `wkhtmltopdf`, which is suppose to show full description in pdf report.